src/core/Slider/component.tsx

```diff
@@ -1,110 +1,109 @@
-import React, { CSSProperties, ReactNode, useEffect, useRef } from "react";
-
+import React, { useState, useEffect, useRef, ReactNode } from "react";
 import Icon from "../Icon/component.tsx";
-import SliderScripts from "./component.js";
 import "./component.css";
 
-type SliderProps = {
-  slides?: ReactNode[];
-  classes?: string;
-  slideClasses?: string;
-  slideMinWidth?: string;
-  slideMaxWidth?: string;
-  mqEnableThreshold?: () => boolean;
+interface SliderProps {
+  children: ReactNode[];
+  interval?: number;
+}
 
-  container?: HTMLDivElement | null;
-};
+function SlideIndicator({
+  numSlides,
+  activeIndex,
+  interval,
+}: {
+  numSlides: number;
```

Comment:
Would be more consistent to have these types be their own interface at the top of the file like `SliderProps`